File: Alarmintro/app/src/main/java/com/example/alarmintro/MainActivity.java
package com.example.alarmintro;

import androidx.appcompat.app.AppCompatActivity;

import android.app.AlarmManager;
import android.app.PendingIntent;
import android.content.Intent;
import android.os.Bundle;
import android.os.SystemClock;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Button btntask= findViewById(R.id.btntask);

        btntask.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent=new Intent(
                        getBaseContext(),
                        MainActivity2.class
                );
                //schedule a task which will run 5 min from now and start my app
                PendingIntent pendingIntent= PendingIntent.getActivity(getBaseContext(),1234,intent,PendingIntent.FLAG_UPDATE_CURRENT);

                AlarmManager alarmManager= (AlarmManager) getSystemService(ALARM_SERVICE);

                //alarmManager.set(AlarmManager.ELAPSED_REALTIME, SystemClock.elapsedRealtime()+60000,pendingIntent);

                alarmManager.setRepeating(AlarmManager.ELAPSED_REALTIME, SystemClock.elapsedRealtime()+60000, 60000,pendingIntent);
               // Intent i2 = new Intent(getBaseContext(),MainActivity.class);

                //PendingIntent pi2= PendingIntent.getActivity(getBaseContext())

            }
        });
    }
}

File: clickablelistview/app/src/main/java/com/example/clickablelistview/MainActivity.kt
package com.example.clickablelistview

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.ArrayAdapter
import android.widget.Toast
import kotlinx.android.synthetic.main.activity_main.*
import kotlinx.android.synthetic.main.list_fruits_items.view.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        listfruits.adapter=ArrayAdapter<String>(
            this,
            R.layout.list_fruits_items,
            R.id.lvfruits,
            arrayOf(
                "apple",
                "mangoes",
                "melons",
                "straws",
                "berrys",
                "bujas",
                "xyzs",
                "guavas",
                "grapes",
                "tuys",
                "papayas"
            )
        )
        listfruits.setOnItemClickListener { parent, view, position, id ->
            Toast.makeText(this,"yash ate ${position+1} ${view.lvfruits.text}",Toast.LENGTH_SHORT).show()
        }

        btn1.setOnClickListener {
            val i =Intent(this,MainActivity2::class.java)
            startActivity(i)
        }
    }
}